financialaid/api_test.py

```diff
@@ -41,7 +41,9 @@ class FinancialAidBaseTestCase(TestCase):
     @classmethod
     def setUpTestData(cls):
         with mute_signals(post_save):
-            cls.profile = ProfileFactory.create()
+            cls.profile = ProfileFactory.create(
+                country="US"
```

Comment:
What does this do? Do we need to set the country to `US` for any of the other profiles created in this function?